packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1182,7 +1185,8 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
 
   void _handleImageFrame(ImageInfo imageInfo, bool synchronousCall) {
     setState(() {
-      _imageInfo = imageInfo;
+      assert(imageInfo.image.width > 0);
```

Comment:
For my own understanding, why this assert?

Reply:
Ahh I added this while testing things to make sure the image wasn't disposed here. I'll replace it with a better assert.

Reply:
Actually just removing it, there's other tests covering this now.